docker/Dockerfile.dev

```diff
@@ -0,0 +1,99 @@
+
+ARG BASE_IMAGE=python:3.10-slim
+FROM $BASE_IMAGE as builder
+
+ARG DEBIAN_FRONTEND="noninteractive"
+
+RUN apt-get update && \
+    apt-get install -q -y --no-install-recommends \
+    build-essential \
+    gcc \
+    libsqlite3-dev \
+    unattended-upgrades && \
+    unattended-upgrade -v
+
+WORKDIR /app
+
+COPY .. .
```

Comment:
What is this copying? It doesn't seem like you have much here yet, or is all of root?

Reply:
It's copying the directory above this one (i.e. yes, it's copying the root). Normally I'd put all the Docker stuff in the project root, so it would be

`COPY . .`

As I didn't know if there'd be much interest in Docker, I tidied it into a sub-directiory.